test/reflection.jl

```diff
@@ -213,3 +213,7 @@ let t13464 = "hey there sailor"
         @test startswith(err13464.msg, "expression is not a function call, or is too complex")
     end
 end
+
+# Issue 14173
+A = randn(2000, 2000)
+@test @elapsed(whos(IOBuffer())) < 5
```

Comment:
how much margin is there here? this might be a bit discriminatory against embedded architectures